templates/docs/examples/templates/maas-layout.html

```diff
@@ -95,23 +95,23 @@ <h1 class="p-heading--3">Machines</h1>
     <div class="col-3">
       <div class="">
         <h4 class="u-sv-1">Filter by</h4>
-        <aside class="p-accordion" role="tablist" aria-multiselect="true">
+        <aside class="p-accordion" role="tablist" >
           <ul class="p-accordion__list">
             <li class="p-accordion__group">
-              <button class="p-accordion__tab" id="owner-tab" role="tab" aria-controls="#owner" aria-expanded="false">Owner</button>
-              <section class="p-accordion__panel" id="owner" role="tabpanel" aria-hidden="true" aria-labelledby="owner-tab">
+              <button class="p-accordion__tab" id="owner" role="region" aria-controls="#owner" aria-expanded="false">Owner</button>
```

Comment:
You changed the id (removed the `-tab`, and now the button has the same id as the section it triggers).
We should not change the ids, or if we want to remove the `tab` from id, we should make it `owner-trigger` or something like that.